src/signals/settings/categories/components/IconInput/IconInput.tsx

```diff
@@ -53,10 +54,14 @@ export const IconInput = ({ formMethods, icon }: Props) => {
 
   const handleOnChange = useCallback(
     async (files: File[]) => {
-      const confirmed = await isConfirmed(
-        'Let op! Je verander een icoon. ',
-        'Er wordt geen back-up van het icoon gemaakt.'
-      )
+      let confirmed
+      if (!fileDataURL) confirmed = true
```

Comment:
in what situation will filedataurl be empty but files not?

Reply:
When you are adding a new icon (as opposed to an existing icon) then fileDataURL will be null, but file will have data. .